www/assets/locales/locale-pl.json

```diff
@@ -262,7 +262,8 @@
       "wifi-thumbnails": "Pokazuj miniatury tylko przez Wi-Fi",
       "show-all-nutriments": "Pokaż wszystkie składniki odżywcze na stronie podglądu",
       "show-units": "Pokaż jednostki odżywcze",
-      "prompt-add-items": "Pytaj o ilość podczas dodawania produktów"
+      "prompt-add-items": "Pytaj o ilość podczas dodawania produktów",
+      "show-macro-elements-summary": "Pokazuj podsumowanie makroelementów per kategoria"
```

Comment:
Do not add translations as part of the pull request. Only add the new locale strings to the `locale-en.json` file. Other languages will be handeled later via Crowdin.